toto.py

```diff
@@ -0,0 +1 @@
+import os
```

Comment:
```suggestion
import sys
```

I think we need sys for this one